code/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="utf-8" />
     <title>Project Name</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
-    </header>
+      
+      <h1>Amy Brown: Medieval Literatures, Modern Cultures</h1>
+      <img src="header-image-crop.jpg" class="headerimage" alt="a red cushioned chair holding books on Old and Middle English literature and medieval sexuality">
+    
+      </header>
 
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
+    <main class="parent">
+      <div class="researchcard">
```

Comment:
You can name the classes with `kebab-case`, so research-card, job-card etc. Read more in code guidelines :)